lib/rack/query_parser.rb

```diff
@@ -177,22 +177,42 @@ def key?(key)
         @params.key?(key)
       end
 
-      def to_params_hash
-        hash = @params
```

Comment:
I found this assignment mostly misleading. It seems to suggest that `hash` is a copy of `@params`, when in reality it is a reference.